### PR TITLE
Added paddle frontend softmax with cross entropy loss

### LINKED ISSUE #22909: softmax_with_cross_entropy

### PR DESCRIPTION
**Pull request by BebehCodes**
Added softmax with cross entropy loss to paddle frontend. 

Close #22909

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Hello, good job on having all the tests passing, but I got a small suggestion below.

**Comment by BebehCodes** [comment]
Hey @Daniel4078 , thanks for pointing it out. I handled it and I believe it looks good now. Have a look and tell me what you think!
Thanks!
